Detail host-agent configuration options (#21)
optimyze/prodfiler-documentation: feature-reference.md

```

* Disables TLS encryption for the communication with collection-agent: set it _only_ if you know you need to send data in
  an unencrypted, non-default location.

```
  -no-kernel-version-check
        Disable checking kernel version for eBPF support. Use at your own risk, to run the agent on older kernels with backported eBPF features.
```

* Allows experimentally running the host-agent on unsupported kernels, disabling the kernel version check.

```
  -project-id int
        The project ID to store the data in.
```

* The project ID where profiles fetched by the host-agent will be stored; instructions in the UI will fill it for you.

```
  -secret-token string
        The secret token associated with the project id.
```

* The secret token to authenticate with the project ID where profiles will be stored; instructions in the UI will fill
  it for you.

```
  -tags string
        User-specified tags separated by ';'. Each tag should match '^[a-zA-Z0-9-:._]+$'.
```

* User-provided tags to mark profiles for a given host: these tags can be used to filter data in the UI as mentioned
  in [filtering](filtering.md#host-derived-keys).

```

>   -t string
>         Shorthand for -tracers. (default "all")

Reviewer: For the purpose of this documentation I think it can be confusing to have short options for some configuration options, but not for all. So I suggest to remove it here.
```suggestion
```

  -tracers string
        Comma-separated list of tracers to include. (default "all")
```

* The tracers will process stacktraces for various languages: we recommend to use `all` to have a holistic view of your
  applications, with stack traces spanning from user to kernel-space.

  Valid tracers are:
    * `native`: native code, such as C/C++/Rust/Go (includes kernel frames)
    * `hotspot`: JVM frames (requires `native` to be included in the list)
    * `python`: Python
    * `php`: PHP
    * `ruby`: Ruby
    * `perl`: Perl

  More languages will be supported soon!

```
  -upload-symbols
        Enables automatic uploading of Golang symbols to the Prodfiler service, for symbolization.
```

> * **Currently supported only for Go binaries**. Upload symbols automatically to the backend: when enabled, you are only
>   required to upload symbols for Go binaries if they are stripped, see [above](#dealing-with-missing-symbols).

Reviewer: This sounds confusing to me. To be more clear we should state that we only upload executables that we have never seen before.
```suggestion
* Automatic upload of parts of unknown executables to support symbolizing these frames. **Currently supported only for Go executables**.
```

Reviewer: external applications is not correct in this context. By unknown I mean executables (to this point limited to Go executables) that our backend did not symbolize before. This includes every executable and is not limited by CI or other builds.

```

>   -v	
>       Shorthand for -verbose.

Reviewer: Same argument as for the short option for `-tracers`.
```suggestion
```

  -verbose
      Enable verbose logging and debugging capabilities.
```

> * Log in verbose mode: set it _only_ when the host-agent fails to start or no data appears in the UI. 

Reviewer: Maybe we should extend this explanation a bit. `[..] or no data appears in the UI` sounds to me as it works like in real time. While our processing is quite fast, I think it can take up to 5 minutes after a successful start till the first data shows up in the UI.
